pulsar-common/src/main/java/org/apache/pulsar/common/util/FutureUtil.java

```diff
@@ -69,6 +70,32 @@ public static <T> CompletableFuture<List<T>> waitForAll(Stream<CompletableFuture
                 })));
     }
 
+    /**
+     * Make {@param dest} completed as {@param src}.
+     */
+    public static <T> void completeAs(final CompletableFuture<T> dest, CompletableFuture<T> src){
+        src.whenComplete((v, ex) -> {
+            if (ex != null){
+                dest.completeExceptionally(ex);
+            } else {
+                dest.complete(v);
+            }
+        });
+    }
+
+    /**
+     * Make {@param dest} completed as {@param src}.
+     */
+    public static <T> void completeAs(final CompletableFuture<Void> dest, CompletableFuture<T>...src){
```

Comment:
Add a better description. rename this method to `completeAfterAll`.

Reply:
Renamed to `completeAfter`,  and modified the description.